caterva2/services/srv_utils.py

```diff
@@ -96,8 +96,11 @@ def reformat_cparams(cparams):
     return cparams
 
 
-def get_relpath(ndarr, cache, scratch):
+def get_relpath(ndarr, cache, scratch, shared):
     path = pathlib.Path(ndarr.schunk.urlpath)
+    if shared is not None and str(path).startswith(str(shared)):
```

Comment:
Personally I prefer to use the pathlib API:

    if shared is not None and path.is_relative_to(shared):

Should work, if `path` is a Path object.

Reply:
done